src/helm/benchmark/presentation/run_specs.conf

```diff
@@ -519,12 +519,12 @@ entries: [
 
   # Data processing
 
-  {description: "entity_matching:model=text,dataset=Beer", priority: 2}
-  {description: "entity_matching:model=text,dataset=Abt_Buy", priority: 2}
-  {description: "entity_matching:model=text,dataset=Dirty_iTunes_Amazon", priority: 2}
+  {description: "entity_matching:model=text,dataset=Beer", priority: 1}
+  {description: "entity_matching:model=text,dataset=Abt_Buy", priority: 1}
+  {description: "entity_matching:model=text,dataset=Dirty_iTunes_Amazon", priority: 1}
 
-  {description: "entity_data_imputation:model=text,dataset=Buy", priority: 2}
-  {description: "entity_data_imputation:model=text,dataset=Restaurant", priority: 2}
+  {description: "entity_data_imputation:model=text,dataset=Buy", priority: 1}
+  {description: "entity_data_imputation:model=text,dataset=Restaurant", priority: 1}
 
   # Code
   {description: "code:model=code,dataset=humaneval", priority: 1}
```

Comment:
I would/might demote code (both APPS and HumanEval) to priority 2.

Reply:
Done.